### PR TITLE
A tutorial for Go clients exists on conjur.org

### LINKED ISSUE #293: A tutorial for Go clients exists on conjur.org

### PR DESCRIPTION
**Pull request by jtuttle**
Closes #293

### CODE REVIEW

File: docs/tutorials/integrations/go.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+The [Conjur API for Go](https://github.com/cyberark/conjur-api-go) provides a programmatic interface to Conjur. You can use the Go API to authenticate with Conjur, load policies, and fetch secrets.
+
+{% include toc.md key='prerequisites' %}
+
+* A [Conjur server](/get-started/) endpoint.
+* The [Conjur API for Go](https://github.com/cyberark/conjur-api-go)
+
+{% include toc.md key='setup' %}
+
+Clone the GitHub repository or use your Golang dependency manager of choice.
```
Comment (dustinmm80) [inline docs/tutorials/integrations/go.md]: We should have a code block following this

```sh-session
$ go get -u cyberark/conjur-api-go
```

File: docs/tutorials/integrations/go.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+conjur, err := conjurapi.NewClientFromKey(
+    config: config, 
+    Login:  os.Getenv("CONJUR_AUTHN_LOGIN"),
+    APIKey: os.Getenv("CONJUR_AUTHN_API_KEY"),
+)
+```
+
+{% include toc.md key='secret-retrieval' %}
+
+Authenicated clients are able to retrieve secrets:
```
Comment (dustinmm80) [inline docs/tutorials/integrations/go.md]: Authenicated -> Authenticated

File: docs/tutorials/integrations/go.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+secretValue, err := conjur.RetrieveSecret(variableIdentifier)
+if err != nil {
+// error handling
+}
+// do something with the secretValue
```
Comment (dustinmm80) [inline docs/tutorials/integrations/go.md]: Can you put something more real-world here? This is a tutorial right, where's the story?

**Comment by doodlesbykumbi** [comment]
@jtuttle looks like you've pulled in the old README. there was a pending PR with a better README and fixed typos. I've gone ahead and merged it. hope it helps https://github.com/cyberark/conjur-api-go#quick-start

**Pull request closed without merging**